manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ZebraTechnologiesCorporation.ZebraDesigner
+PackageVersion: 2.5.0.9420
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Designer/zebradesigner-v250-9420.exe

```

Comment:
```suggestion
- InstallerUrl: https://www.zebra.com/apps/dlmanager?dlp=7b8a235653193b4c72c440110c25661656f56f5180957c98e7c0bc2144149cd156a1bc6e684725abae8eaa3b64ee10902b6e77e74d8082bcecf0ffbe929cfe5e5810126dd5b9b88a5fcc3c78383a5eaf7ad78c349cbcb259f5555a613f845da395cb92691dcef46172cce19429d4f28cc2105744ba5f4ca7&c=us&l=en
```

Reply:
I'm not sure that this URL will work for other users, as it appears to use a unique key. Also the website is designed to require EULA agreement before processing the download for older versions. For the current version 3.0 there appear to be fewer download protections.